src/security/templatetags/securitytags.py

```diff
@@ -69,6 +69,12 @@ def is_author(context):
     return request.user.is_author(request)
 
 
+@register.simple_tag(takes_context=True)
+def is_repository_manager(context):
+    request = context['request']
+    return request.user.is_repository_manager(request)
```

Comment:
If you make the change I proposed earlier, then we need:
`return request.user.is_repository_manager(request.repository)`